.travis.yml

```diff
@@ -24,8 +25,17 @@ android:
   - platform-tools
   - tools
   - android-22
-  - sys-img-armeabi-v7a-android-22
-  
+  - extra-google-google_play_services
+  - extra-google-m2repository
+  - extra-android-m2repository
+  - extra-android-support
+  - addon-google_apis-google-22 # Google play services
+  - sys-img-armeabi-v7a-google_apis-22
+
+licenses:
```

Comment:
"By default, Travis CI will accept all the requested licenses, but it is also possible to define a white list of licenses to be accepted, as shown in the following example:"
from https://docs.travis-ci.com/user/languages/android/#Pre-installed-components

Reply:
You're right, we most likely do not need the license list. I'll remove it, once I'm sure, that the build succeeds :)